deploy/crd.yaml

```diff
@@ -30,6 +30,33 @@ spec:
     kind: Database
```

Comment:
Add validation for superUser, superUserPassword, DBHost

Reply:
done